src/flag_gems/ops/normal.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import torch
+import triton
+
+from ..utils import pointwise_dynamic
+from ..utils.random_utils import philox_cuda_seed_offset
+from ..utils.shape_utils import broadcast_shapes, volume
+from .randn import randn_kernel
+
+UNROLL = 4
+
+
+@pointwise_dynamic(
+    is_tensor=[True, True, True], promotion_methods=[(0, 1, 2, "DEFAULT")]
+)
+@triton.jit
+def transform_func_tensor_tensor(val, std, mean):
+    return val * std + mean
+
+
+@pointwise_dynamic(
+    is_tensor=[True, True, False], promotion_methods=[(0, 1, 2, "DEFAULT")]
+)
+@triton.jit
+def transform_func_tensor_float(val, std, mean):
+    return val * std + mean
+
+
+@pointwise_dynamic(
+    is_tensor=[True, False, True], promotion_methods=[(0, 1, 2, "DEFAULT")]
+)
+@triton.jit
+def transform_func_float_tensor(val, std, mean):
+    return val * std + mean
+
+
+@pointwise_dynamic(
+    is_tensor=[True, False, False], promotion_methods=[(0, 1, 2, "DEFAULT")]
+)
+@triton.jit
+def transform_func_float_float(val, std, mean):
```

Comment:
The function name is not informative.

Reply:
This name consist with aten name `tensor_tenosr`_normal, `tensor_float_normal`, `float_float_normal`